packages/site/src/ui/link.js

```diff
@@ -1,8 +1,9 @@
-import DSLink from '@pluralsight/ps-design-system-link'
 import NextLink from 'next/link.js'
 import PropTypes from 'prop-types'
 import React from 'react'
 
+import DSLink from '@pluralsight/ps-design-system-link'
```

Comment:
Named imports all belong in one block in my personal opinion, whether we wrote them or not.  I believe the project started and is still mostly that way. We could talk as a team to adjust this.

Reply:
yeah we should talk. I know we talked about it in the past and maybe we didn't lock down a final decision. i've generally been organizing in this fashion but I might have been doing that out of turn

Reply:
Definitely not out of turn in terms of it not being allowed or sanctioned or whatever.  I think our styleguide is too ambiguous for that.  2 weeks in the brig should be sufficient. :)